include/llbuild/Basic/FileSystem.h

```diff
@@ -85,6 +85,57 @@ class FileSystem  {
 /// Create a FileSystem instance suitable for accessing the local filesystem.
 std::unique_ptr<FileSystem> createLocalFileSystem();
 
+
+// Device/inode agnostic filesystem wrapper
```

Comment:
By convention, we typically use `///` at the start of classes (doxygen-style) to document them. We don't actually have any doxygen generation wired up, but I try and still follow the convention.